### DISCUSSION TITLE
Live Flight Info

### DISCUSSION THREAD

**Discussion by jameslashmar** [johanohly/AirTrail#29] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great if adding a flight in the future, to be able to have live flight updates & notifications

**Comment by johanohly** [comment]
This is definitely out of scope for this project. 
Even if I wanted to implement this, I wouldn't be able to, as data sources for live flight data, especially for the amount required for this feature, cost money.
